iosApp/iosApp/Pages/Map/MapLayerManager.swift

```diff
@@ -76,9 +83,9 @@ class MapLayerManager: IMapLayerManager {
                 }
 
                 if map.layerExists(withId: "puck") {
-                    try map.addLayer(layer, layerPosition: .below("puck"))
+                    try map.addPersistentLayer(layer, layerPosition: .below("puck"))
```

Comment:
persistentLayer keeps the map layers from flashing in every time the map returns from background